Header header

pair_array[] endpoints
pair_array[] radius_values
pair_array[] alpha_values
int64 num_lines

